commands/music.py

```diff
@@ -319,53 +325,67 @@ async def _leave(self, ctx: commands.Context):
         await ctx.voice_state.stop()
         del self.voice_states[ctx.guild.id]
 
-    @commands.command(name='volume')
-    async def _volume(self, ctx: commands.Context, *, volume: int):
-        """Sets the volume of the player."""
+    @commands.command(name='volume', aliases=['vol'])
+    async def change_volume(self, ctx, *, vol: float):
```

Comment:
I thought this didn't have any affect since users can set volume locally, and was being removed. Has this changed, or does it work now?

Reply:
It didn't seem to work even after changing it. I decided to leave it there to see if we would try to get it working or just abandon it. I personally think it's unnecessary to have and am with the idea of removing it.

Reply:
You're running this feature implementation. If it doesn't do anything and you're in favor of taking it out, I don't see any reason to keep it. Particularly since it isn't needed anyways.